nilearn/plotting/displays.py

```diff
@@ -526,11 +526,8 @@ def _map_show(self, img, type='imshow',
 
         if threshold is not None:
             data = img.get_data()
-            if threshold == 0:
-                data = np.ma.masked_equal(data, 0, copy=False)
-            else:
-                data = np.ma.masked_inside(data, -threshold, threshold,
-                                           copy=False)
+            mask = np.abs(data) <= threshold
```

Comment:
Small remark: the problem is not the function `masked_equal` itself but the fact that we do `-threshold` on an unisgned value. I'm afraid that this gag will happen here and there in the code (typically in the colorbar code). That's only a supposition but maybe we should try to find a solution and propagate it through the code?